src/Stackla/Api/Tile.php

```diff
@@ -627,7 +627,7 @@ public function getByGuid($id)
 
     public function getByStacklaFeedId($id)
     {
-        return parent::getById('sta_feed_id:'.$id);
+        return parent::getById('guid:'.$id);
```

Comment:
Hi @Pixadelic, `sta_feed_id` and `guid` are two different fields, it would better to have it in different method instead of changing the `sta_feed_id` to `guid`